<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoCapture_Gui</class>
 <widget class="QWidget" name="VideoCapture_Gui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Start_pushButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(66, 69, 183);
background-color: green;
border-width: 3px;        
border-style: solid;
border-radius: 50px;</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Stop_pushButton">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-color: rgb(66, 69, 183);
background-color: red;
border-width: 3px;        
border-style: solid;
border-radius: 50px;</string>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QLabel" name="Video_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>854</width>
     <height>480</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="FPS_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#f80000;&quot;&gt;None&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
